### ISSUE TITLE
Enhance PHPUnit version definitions in composer.json

### ISSUE THREAD

**Issue by peter279k** [php-amqplib/php-amqplib#718]
According to the [composer.json](https://github.com/php-amqplib/php-amqplib/blob/master/composer.json), we can find the following dependency definitions inside `require` block:

```
......
   "require": {
        "php": ">=5.6",
        "ext-bcmath": "*",
        "ext-sockets": "*"
    },
    "require-dev": {
        "ext-curl": "*",
        "phpunit/phpunit": "^4.8|^5.7",
        "squizlabs/php_codesniffer": "^2.5",
        "phpdocumentor/phpdocumentor": "^2.9",
        "nategood/httpful": "^0.2.20"
    },
......
```

We can find that the `phpunit/phpunit` definition should be enhanced.

The minimum PHP version requires `php-5.6`.
And the PHPUnit version requires `^5.7` at least.
This requirement message is mentioned on official [PHPUnit site](https://phpunit.de/getting-started/phpunit-5.html).

We can change into ` "phpunit/phpunit": "^5.7",` definition.

**Comment by ramunasd** [comment]
@peter279k please feel free to sumit PR for this, it is really simple issue.

**Comment by peter279k** [comment]
Thanks for your concern, @ramunasd.

I will create PR for this.

**Comment by ramunasd** [comment]
please fix phpunit.xml.dist also, there is some deprecated features and attributes..

**Comment by lukebakken** [comment]
Fixed by #722
